docs/DeveloperGuide.md

```diff
@@ -579,17 +605,17 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Deleting an application
 
-1. Deleting a person while all persons are being shown
+1. Deleting an application while all applications are being shown
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Prerequisites: List all applications using the `list` command. Multiple applications in the list.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First application is deleted from the list. Details of the deleted application shown in the status message. Timestamp in the status bar is updated.
```

Comment:
actually what is the `timestamp in the status bar` referring to? correct me if I am wrong but there's no such time stamp on the status bar right or am I looking at the wrong thing

Reply:
Oh yeah ps I didn't notice it, thanks for pointing out! 😅